test/ctia/stores/es/crud_test.clj

```diff
@@ -377,7 +377,17 @@
                                        :granularity :month
                                        :aggs {:agg-type :cardinality
                                               :agg-key :nb-orgs
-                                              :aggregate-on "groups"}}}))))
+                                              :aggregate-on "groups"}}})))
+  (is (= {:metric
+          {:date_histogram
+           {:field     "title.whole"
+            :interval  :week
+            :time_zone "+02:00"}}}
```

Comment:
you need an example with topn and another with cardinality but not histogram :-)